syntax = "proto3";

package routesia.rpc;


message RPCError {
    // Response codes
    //
    enum ResponseCode {
        UNSPECIFIED_ERROR = 0;
        HANDLER_NOT_FOUND = 1;
        ENTITY_NOT_FOUND = 2;
    }
    ResponseCode response_code = 1;

    // Message will always be a string
    //
    string message = 2;
}
